plugins/connectrpc/query-es/v2.0.2/buf.plugin.yaml

```diff
@@ -0,0 +1,22 @@
+version: v1
+name: buf.build/connectrpc/query-es
+plugin_version: v2.0.2
+source_url: https://github.com/connectrpc/connect-query-es
+integration_guide_url: https://connectrpc.com/docs/web/query/getting-started
+description: Generates stubs for calling Connect RPCs with TanStack Query.
+deps:
+  - plugin: buf.build/bufbuild/es:v2.2.3
+output_languages:
+  - javascript
+  - typescript
+registry:
+  opts:
+    - import_extension=js
+  npm:
+    import_style: module
+    rewrite_import_path_suffix: connectquery.js
+    deps:
+      - package: '@bufbuild/protobuf'
+        version: ^2.2.3
```

Comment:
The only real change in this version of the plugin:
- Added import_extension=js
- Removed dep on @connectrpc/connect-query